collections/nemo_nlp/nemo_nlp/data/data_layers.py

```diff
@@ -17,20 +17,22 @@
            'BertPretrainingPreprocessedDataLayer',
            'TranslationDataLayer',
            'GlueDataLayerClassification',
-           'GlueDataLayerRegression']
+           'GlueDataLayerRegression',
+           'BertQuestionAnsweringDataLayer']
 
 # from abc import abstractmethod
 import sys
```

Comment:
Can you clean up these imports too?

Reply:
```
import sys
import os
import random

import numpy as np
import h5py
import torch
from torch.utils import data as pt_data
```
Can you make these in alphabetical order (os, random, sys) and (h5py, numpy, torch)?